app/static/styles/main.css

```diff
@@ -49,17 +66,73 @@ body {
     padding-right: 0px;
 }
 
+.nav-icon {
+    display: none;
+    cursor: pointer;
+    position: fixed;
+    right: 5%;
+    z-index: 100;
+}
+
+.nav-icon i {
+    color: white;
+    font-size: 1.5em;
+}
+
+.nav-menu {
+    list-style: none;
+    font-family: 'Inconsolata', monospace;
```

Comment:
For consistency's sake, why don't we go back to using the same font universally? I feel like it would be better for coherency.

Reply:
I just took some inspiration from the MLH Fellowship page's navbar, that's why I used a different font, would you like me to use the same font for consistency?